optimize/client/src/modules/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {useEffect, useState} from 'react';
+import {Link, useLocation} from 'react-router-dom';
+import {Breadcrumb, BreadcrumbItem, BreadcrumbSkeleton} from '@carbon/react';
+
+import {useErrorHandling} from 'hooks';
+import {showError} from 'notifications';
+import {t} from 'translation';
+
+import {loadEntitiesNames, EntityNamesResponse, getEntityId} from './service';
+
+import './Breadcrumbs.scss';
+
+export default function Breadcrumbs() {
+  const {mightFail} = useErrorHandling();
+  const location = useLocation();
+  const [entityNames, setEntityNames] = useState<EntityNamesResponse>();
+  const collection = getEntityId('collection', location.pathname);
+  const dashboard = getEntityId('dashboard', location.pathname);
```

Comment:
Why can't we use something like `useMatches` from `react-router-dom`?

Reply:
We are still using react-router v5 which does not have `useMatches`. We can however use `useRouteMatch` but i find using it makes the solution more complicated.